src/backend/distributed/sql/downgrades/citus--11.1-1--11.0-4.sql

```diff
@@ -86,6 +86,7 @@ DROP FUNCTION pg_catalog.get_all_active_transactions(OUT datid oid, OUT process_
                                                      OUT worker_query BOOL, OUT transaction_number int8, OUT transaction_stamp timestamptz,
                                                      OUT global_pid int8);
 #include "../udfs/get_all_active_transactions/11.0-1.sql"
+#include "../udfs/citus_finish_pg_upgrade/11.0-1.sql"
```

Comment:
The end of this downgrade is 11.0-4 and I expect to see the same definition of citus_finish_pg_upgrade. However, the earlier version of this UDF should be installed on the other downgrade script in here, namely `citus--11.0-4--11.0-3.sql`